packages/blade/src/components/Input/DropdownInputTriggers/__tests__/AutoComplete.ssr.test.tsx

```diff
@@ -3,7 +3,7 @@ import userEvent from '@testing-library/user-event';
 import { waitFor } from '@testing-library/react';
 import { Dropdown, DropdownOverlay } from '~components/Dropdown';
 import { DropdownFooter, DropdownHeader } from '~components/Dropdown/DropdownHeaderFooter';
-import renderWithSSR from '~utils/testing/renderWithSSR';
+import renderWithSSR from '~utils/testing/renderWithSSR.web';
```

Comment:
Why .web? It should take the right extension anyway right?

Reply:
nope it seems to fail on ssr storybook. This is how we seem to do in all ssr files.